assets/src/edit-story/stickers/index.js

```diff
@@ -18,7 +18,10 @@
  */
 import { default as sample } from './sampleSticker';
 import { default as beautyHeart } from './beautyHeart';
+import * as diyStickers from './diy';
+
 export default {
   sample,
   beautyHeart,
+  ...diyStickers,
```

Comment:
yea I like how much this cleans it up. especially once we got a bunch of new stickers in here for the templates should make all the stickers much more organized 🎉